packages/dataviews/src/dataviews.js

```diff
@@ -28,15 +29,34 @@ export default function DataViews( {
 	actions,
 	data,
 	getItemId = defaultGetItemId,
+	getItemTitle,
```

Comment:
I feel this prop is field API territory: its purpose is telling dataviews what is the _field_ that we should use for announcing selection. It's doing what `getValue`/`render` should be doing. Knowing what's the "field that communicates the title" is something we need, and not only for bulk selection.

Could we make this explicit and model it as part of the field API? We could leverage the `primaryField`, for instance:

- make the table layout declare its primary field: it just happens to be `title` in pages & templates, but it doesn't have to.
- update the code that uses `getItemTitle` to use `primaryField.getValue` instead.

Reply:
Hi @oandregal, I updated the code to use primaryField.getValue, and removed getItemTitle.